src/pocketdb/repositories/TransactionRepository.cpp

```diff
@@ -459,6 +459,29 @@ namespace PocketDb
         });
     }
 
+    // Remove all transactions and PocketNet payloads which are older than the specified time
+    // and have not yet been added to a block 
+    void TransactionRepository::CleanExpiredTransactions(int64_t time)
+    {
+        std::vector<std::string> hashes;
+        TryTransactionStep(__func__, [&]()
+        {
+
+            // Get list of expired transaction hashes which have not been added to a block
+            auto stmt = SetupSqlStatement(R"sql(
+                select Hash, Time from Transactions t where t.Hash isnull and t.Height isnull and t.Time < ?
```

Comment:
I think there's a typo here, meaning `t.BlockHash isnull`?

Reply:
This code needs some more work.  I will remove it from this patch an resubmit.